Automattic/wp-calypso
# State: Cleanup previous JPO implementation

This PR cleans up the previous Redux implementation of JPO that we're not going to use. This includes removal of unused:

* Action types
* Action creators
* Reducers & schemas
* Selectors
* Tests for the above

To test:

* Verify all we're removing is indeed unused (`grep` to the rescue)
* Verify all tests still pass (`npm run test-client`)
